CHANGELOG.md

```diff
@@ -62,6 +62,7 @@ All notable changes to this project will be documented in this file.
 -   #1892 : Fix implementation of list function when an iterable is passed as parameter.
 -   #1924 : Fix internal error arising in Duplicate or list comprehensions.
 -   #1970 : Fix missing `TypeError` for wrong type passed as optional argument.
+-   #1985 : Fix implementation of gcd and lcm for C and Fortran.
```

Comment:
Function names should be in code quotes

Reply:
Done [4dbf41c](https://github.com/pyccel/pyccel/pull/1986/commits/4dbf41c7fb7a2744fe8a10314cee2d5fcced65a7)